src/apps/authentication/__test__/components/UserOrganisationDetails.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import {render} from "../../../../test-utils";
+import {UserOrganisationDetails} from "../../components";
+import {testAPIOrgList} from "../test_data";
+
+type userOrganisationDetailsProps = React.ComponentProps<typeof UserOrganisationDetails>;
+
+const baseProps: userOrganisationDetailsProps = {
+    orgs: testAPIOrgList
+};
+
+function renderUI(props: Partial<userOrganisationDetailsProps> = {}) {
+    return render(<UserOrganisationDetails {...baseProps} {...props} />);
+}
+
+describe("UserOrganisationDetails", () => {
+    it("should match snapshot", () => {
+        const {container} = renderUI(baseProps);
+
+        expect(container).toMatchSnapshot();
+    });
+});
+
+describe("View User Organisations", () => {
+    it("should not show no user organisations message", () => {
```

Comment:
@tarunkumar-tw @supriyatw modify it statement like this `should not show no user organisations message when user is part of at least one org`